Tests/UIPluginTests.swift

```diff
@@ -5,45 +5,21 @@ import Nimble
 class UIPluginTests: QuickSpec {
 
     override func spec() {
-        describe("Instantiation") {
+        describe("UIPlugin") {
```

Comment:
Following the project conventions, the Class name on describe must have the prefix: ``.``

Reply:
We use `.` for class methods (aka `static`) and `#` for instance methods. The class name is neither of these ones... what do you think?

Reply:
;0I ended up confusing the conventions so you can disregard my comment :)